src/widgets/WalletModal/CopyToClipboard.tsx

```diff
@@ -30,17 +30,33 @@ const Tooltip = styled.div<{ isTooltipDisplayed: boolean }>`
 const CopyToClipboard: React.FC<Props> = ({ toCopy, children, ...props }) => {
   const [isTooltipDisplayed, setIsTooltipDisplayed] = useState(false);
 
+  const copyToClipboard = (content: string) => {
+    const el = document.createElement("textarea");
```

Comment:
This is a nitpick - but we could perhaps use `input` instead of `textarea`. Even though in 99% of the cases it is too quick to notice anything - in case where user has super slow computer they might notice something in theory and `input` just takes less space on the screen.